VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -288,8 +284,10 @@ public void onNegativeClicked() {
                 MediaConsentUtils.consentToMediaPlayback(getActivity(), dialogCallback,
                         environment.getConfig());
             } else {
-                //Video is downloaded. Hence play
-                startOnlinePlay(entry);
+                if (playerFragment != null && !playerFragment.isFrozen()) {
```

Comment:
`setUserVisibleHint()` is called before `onActivityCreated()`. So, null check is valid here.